### ISSUE TITLE
Value Error when setting years

### ISSUE THREAD

**Issue by schwarzan** [dr-prodigy/python-holidays#527]
Since the ugrade to version '0.11.3' I have encountered the following error:

```
In [1]: import holidays
   ...: print(holidays.__version__)
   ...: years = array([2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021])
   ...: holidays.CZ(years=years)
Out[1]: 
'0.11.3'
*** ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
```

Using the previous version I do not encounter this error.

Thank you so much for your awesome library!

**Comment by mborsetti** [comment]
I cannot reproduce this error:

```
import holidays
print(holidays.__version__)
years = array([2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021])
holidays.CZ(years=years)
0.11.3
Traceback (most recent call last):
  File "C:\Program Files\Python\Python39\lib\site-packages\IPython\core\interactiveshell.py", line 3444, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-6-e5c753ba3e41>", line 3, in <module>
    years = array([2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021])
TypeError: 'module' object is not callable
```

**Comment by mborsetti** [comment]
Hmmm, I cannot reproduce the error:
```
import holidays
print(holidays.__version__)
years = array([2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021])
holidays.CZ(years=years)
0.11.3
Traceback (most recent call last):
  File "C:\Program Files\Python\Python39\lib\site-packages\IPython\core\interactiveshell.py", line 3444, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-2-e5c753ba3e41>", line 3, in <module>
    years = array([2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021])
NameError: name 'array' is not defined
```

However, this works as advertised:
```
import holidays
print(holidays.__version__)
years = [2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021]
holidays.CZ(years=years)
0.11.3
Out[4]: 
{datetime.date(2016, 1, 1): 'Den obnovy samostatného českého státu',
 datetime.date(2016, 3, 25): 'Velký pátek',
 datetime.date(2016, 3, 28): 'Velikonoční pondělí',
 datetime.date(2016, 5, 1): 'Svátek práce',
 datetime.date(2016, 5, 8): 'Den vítězství',
 datetime.date(2016, 7, 5): 'Den slovanských věrozvěstů Cyrila a Metoděje',
 datetime.date(2016, 7, 6): 'Den upálení mistra Jana Husa',
 datetime.date(2016, 9, 28): 'Den české státnosti',
 datetime.date(2016, 10, 28): 'Den vzniku samostatného československého státu',
 datetime.date(2016, 11, 17): 'Den boje za svobodu a demokracii',
 datetime.date(2016, 12, 24): 'Štědrý den',
 datetime.date(2016, 12, 25): '1. svátek vánoční',
 datetime.date(2016, 12, 26): '2. svátek vánoční',
 datetime.date(2017, 1, 1): 'Den obnovy samostatného českého státu',
 datetime.date(2017, 4, 14): 'Velký pátek',
 datetime.date(2017, 4, 17): 'Velikonoční pondělí',
 datetime.date(2017, 5, 1): 'Svátek práce',
 datetime.date(2017, 5, 8): 'Den vítězství',
 datetime.date(2017, 7, 5): 'Den slovanských věrozvěstů Cyrila a Metoděje',
 datetime.date(2017, 7, 6): 'Den upálení mistra Jana Husa',
 datetime.date(2017, 9, 28): 'Den české státnosti',
 datetime.date(2017, 10, 28): 'Den vzniku samostatného československého státu',
 datetime.date(2017, 11, 17): 'Den boje za svobodu a demokracii',
 datetime.date(2017, 12, 24): 'Štědrý den',
 datetime.date(2017, 12, 25): '1. svátek vánoční',
 datetime.date(2017, 12, 26): '2. svátek vánoční',
 datetime.date(2018, 1, 1): 'Den obnovy samostatného českého státu',
 datetime.date(2018, 3, 30): 'Velký pátek',
 datetime.date(2018, 4, 2): 'Velikonoční pondělí',
 datetime.date(2018, 5, 1): 'Svátek práce',
 datetime.date(2018, 5, 8): 'Den vítězství',
 datetime.date(2018, 7, 5): 'Den slovanských věrozvěstů Cyrila a Metoděje',
 datetime.date(2018, 7, 6): 'Den upálení mistra Jana Husa',
 datetime.date(2018, 9, 28): 'Den české státnosti',
 datetime.date(2018, 10, 28): 'Den vzniku samostatného československého státu',
 datetime.date(2018, 11, 17): 'Den boje za svobodu a demokracii',
 datetime.date(2018, 12, 24): 'Štědrý den',
 datetime.date(2018, 12, 25): '1. svátek vánoční',
 datetime.date(2018, 12, 26): '2. svátek vánoční',
 datetime.date(2019, 1, 1): 'Den obnovy samostatného českého státu',
 datetime.date(2019, 4, 19): 'Velký pátek',
 datetime.date(2019, 4, 22): 'Velikonoční pondělí',
 datetime.date(2019, 5, 1): 'Svátek práce',
 datetime.date(2019, 5, 8): 'Den vítězství',
 datetime.date(2019, 7, 5): 'Den slovanských věrozvěstů Cyrila a Metoděje',
 datetime.date(2019, 7, 6): 'Den upálení mistra Jana Husa',
 datetime.date(2019, 9, 28): 'Den české státnosti',
 datetime.date(2019, 10, 28): 'Den vzniku samostatného československého státu',
 datetime.date(2019, 11, 17): 'Den boje za svobodu a demokracii',
 datetime.date(2019, 12, 24): 'Štědrý den',
 datetime.date(2019, 12, 25): '1. svátek vánoční',
 datetime.date(2019, 12, 26): '2. svátek vánoční',
 datetime.date(2020, 1, 1): 'Den obnovy samostatného českého státu',
 datetime.date(2020, 4, 10): 'Velký pátek',
 datetime.date(2020, 4, 13): 'Velikonoční pondělí',
 datetime.date(2020, 5, 1): 'Svátek práce',
 datetime.date(2020, 5, 8): 'Den vítězství',
 datetime.date(2020, 7, 5): 'Den slovanských věrozvěstů Cyrila a Metoděje',
 datetime.date(2020, 7, 6): 'Den upálení mistra Jana Husa',
 datetime.date(2020, 9, 28): 'Den české státnosti',
 datetime.date(2020, 10, 28): 'Den vzniku samostatného československého státu',
 datetime.date(2020, 11, 17): 'Den boje za svobodu a demokracii',
 datetime.date(2020, 12, 24): 'Štědrý den',
 datetime.date(2020, 12, 25): '1. svátek vánoční',
 datetime.date(2020, 12, 26): '2. svátek vánoční',
 datetime.date(2021, 1, 1): 'Den obnovy samostatného českého státu',
 datetime.date(2021, 4, 2): 'Velký pátek',
 datetime.date(2021, 4, 5): 'Velikonoční pondělí',
 datetime.date(2021, 5, 1): 'Svátek práce',
 datetime.date(2021, 5, 8): 'Den vítězství',
 datetime.date(2021, 7, 5): 'Den slovanských věrozvěstů Cyrila a Metoděje',
 datetime.date(2021, 7, 6): 'Den upálení mistra Jana Husa',
 datetime.date(2021, 9, 28): 'Den české státnosti',
 datetime.date(2021, 10, 28): 'Den vzniku samostatného československého státu',
 datetime.date(2021, 11, 17): 'Den boje za svobodu a demokracii',
 datetime.date(2021, 12, 24): 'Štědrý den',
 datetime.date(2021, 12, 25): '1. svátek vánoční',
 datetime.date(2021, 12, 26): '2. svátek vánoční',
 datetime.date(2011, 1, 1): 'Den obnovy samostatného českého státu',
 datetime.date(2011, 4, 25): 'Velikonoční pondělí',
 datetime.date(2011, 5, 1): 'Svátek práce',
 datetime.date(2011, 5, 8): 'Den vítězství',
 datetime.date(2011, 7, 5): 'Den slovanských věrozvěstů Cyrila a Metoděje',
 datetime.date(2011, 7, 6): 'Den upálení mistra Jana Husa',
 datetime.date(2011, 9, 28): 'Den české státnosti',
 datetime.date(2011, 10, 28): 'Den vzniku samostatného československého státu',
 datetime.date(2011, 11, 17): 'Den boje za svobodu a demokracii',
 datetime.date(2011, 12, 24): 'Štědrý den',
 datetime.date(2011, 12, 25): '1. svátek vánoční',
 datetime.date(2011, 12, 26): '2. svátek vánoční',
 datetime.date(2012, 1, 1): 'Den obnovy samostatného českého státu',
 datetime.date(2012, 4, 9): 'Velikonoční pondělí',
 datetime.date(2012, 5, 1): 'Svátek práce',
 datetime.date(2012, 5, 8): 'Den vítězství',
 datetime.date(2012, 7, 5): 'Den slovanských věrozvěstů Cyrila a Metoděje',
 datetime.date(2012, 7, 6): 'Den upálení mistra Jana Husa',
 datetime.date(2012, 9, 28): 'Den české státnosti',
 datetime.date(2012, 10, 28): 'Den vzniku samostatného československého státu',
 datetime.date(2012, 11, 17): 'Den boje za svobodu a demokracii',
 datetime.date(2012, 12, 24): 'Štědrý den',
 datetime.date(2012, 12, 25): '1. svátek vánoční',
 datetime.date(2012, 12, 26): '2. svátek vánoční',
 datetime.date(2013, 1, 1): 'Den obnovy samostatného českého státu',
 datetime.date(2013, 4, 1): 'Velikonoční pondělí',
 datetime.date(2013, 5, 1): 'Svátek práce',
 datetime.date(2013, 5, 8): 'Den vítězství',
 datetime.date(2013, 7, 5): 'Den slovanských věrozvěstů Cyrila a Metoděje',
 datetime.date(2013, 7, 6): 'Den upálení mistra Jana Husa',
 datetime.date(2013, 9, 28): 'Den české státnosti',
 datetime.date(2013, 10, 28): 'Den vzniku samostatného československého státu',
 datetime.date(2013, 11, 17): 'Den boje za svobodu a demokracii',
 datetime.date(2013, 12, 24): 'Štědrý den',
 datetime.date(2013, 12, 25): '1. svátek vánoční',
 datetime.date(2013, 12, 26): '2. svátek vánoční',
 datetime.date(2014, 1, 1): 'Den obnovy samostatného českého státu',
 datetime.date(2014, 4, 21): 'Velikonoční pondělí',
 datetime.date(2014, 5, 1): 'Svátek práce',
 datetime.date(2014, 5, 8): 'Den vítězství',
 datetime.date(2014, 7, 5): 'Den slovanských věrozvěstů Cyrila a Metoděje',
 datetime.date(2014, 7, 6): 'Den upálení mistra Jana Husa',
 datetime.date(2014, 9, 28): 'Den české státnosti',
 datetime.date(2014, 10, 28): 'Den vzniku samostatného československého státu',
 datetime.date(2014, 11, 17): 'Den boje za svobodu a demokracii',
 datetime.date(2014, 12, 24): 'Štědrý den',
 datetime.date(2014, 12, 25): '1. svátek vánoční',
 datetime.date(2014, 12, 26): '2. svátek vánoční',
 datetime.date(2015, 1, 1): 'Den obnovy samostatného českého státu',
 datetime.date(2015, 4, 6): 'Velikonoční pondělí',
 datetime.date(2015, 5, 1): 'Svátek práce',
 datetime.date(2015, 5, 8): 'Den vítězství',
 datetime.date(2015, 7, 5): 'Den slovanských věrozvěstů Cyrila a Metoděje',
 datetime.date(2015, 7, 6): 'Den upálení mistra Jana Husa',
 datetime.date(2015, 9, 28): 'Den české státnosti',
 datetime.date(2015, 10, 28): 'Den vzniku samostatného československého státu',
 datetime.date(2015, 11, 17): 'Den boje za svobodu a demokracii',
 datetime.date(2015, 12, 24): 'Štědrý den',
 datetime.date(2015, 12, 25): '1. svátek vánoční',
 datetime.date(2015, 12, 26): '2. svátek vánoční'}
```

**Comment by dr-prodigy** [comment]
The issue should be fixed in #528 , and about to be published as version 0.11.3.1 (v.0.11.3 has instead been deleted from PyPi).
@schwarzan I kindly ask you to confirm that the last release fixes your issue.
Thank you both 👍👍
